darwin/dataset/upload_manager.py

```diff
@@ -162,9 +162,19 @@ def serialize(self):
 
     def serialize_v2(self):
```

Comment:
It's not a change you've made, but this function `serialize_v2` doesn't actually seem to serialize/serialise data, just makes it ready for serialisation?